worldcup/src/components/random.jsx

```diff
@@ -0,0 +1,56 @@
+import { Stage } from "../style/index";
+import { Question, ChoiceBox, ChoiceBoxWrapper, Button } from "../style/item";
+import { useState, useEffect } from "react";
+import RandomResult from "./random-result";
+import styled, { keyframes } from 'styled-components';
+
+function CountDown() {
+    function Timer(){
+        const [time, setTime] = useState(3);
+    
+        useEffect(() => {
+            time > 0 && setTimeout(() => setTime(time - 1), 1000);
+        }, [time]);
+    
+        return time
+        ? <Stage><NumEffect>{time}</NumEffect></Stage>
+        : <RandomResult></RandomResult> ;
+    }
+
+    return (<>{Timer()}</>);
+}
```

Comment:
CountDown 컴포넌트가 하고 있는 역할이 
- Timer 컴포넌트 생성
- Timer 컴포넌트 반환(렌더링)

밖에 없기 때문에, CountDown과 Timer는 불필요하게 중첩되진 않았는지 고민해보면 좋을 것 같아요!! 

즉, 아래와 같이 Timer를 새로운 함수로 정의해주지 않아도 충분히 표현이 가능하겠죠?? 
```suggestion
function CountDown() {
      const [time, setTime] = useState(3);
  
      useEffect(() => {
          time > 0 && setTimeout(() => setTime(time - 1), 1000);
      }, [time]);
  
      return time
      ? <Stage><NumEffect>{time}</NumEffect></Stage>
      : <RandomResult></RandomResult> ;
}
```

또 Stage 컴포넌트를 여기서 사용하는 것이 맞나! 고민해봐도 좋을 것 같아요! 
Stage 컴포넌트가 어딘가에서 재사용되기 때문에 index 파일로 분리시킨 것일 텐데, 
다른 곳에서 Stage 컴포넌트가 보여지는 스타일과, 이곳에서 time 숫자가 보여지는 스타일이 연관성이 전혀 없다면 
아무리 CSS 코드가 동일한 부분이 있더라도, 같은 모듈로 묶어주지 않는 것이 좋을 수도 있어요! 

말이 어려울 수 있는데, 쉽게 말하면 이곳은 `stage` 즉 단계를 보여주는 것과 무관한 `랜덤 추천 타이머`를 보여주는 부분이기 때문에 단계를 보여주는 역할의 컴포넌트가 재사용되기에는 찰떡의 자리는 아닐 수 있다! 라는 의견도 한번 남겨봅니당

Reply:
말 어렵지 않았구 완전 이해했습니다❤❤❤ 승희 말대로 CSS가 동일해서 그렇게 쓴 건데 굳이 이렇게 안 써줘도 되는 부분이었던 것 같아요!! 친절한 조언 완전 감사랑합니다❤

Reply:
숭희 피드백 완전 좋다..... 유익해.....